NitroxClient/Communication/PacketReceiver.cs

```diff
@@ -16,11 +16,11 @@ public PacketReceiver(INetworkDebugger networkDebugger = null)
             this.networkDebugger = networkDebugger;
         }
 
-        public void PacketReceived(Packet packet)
+        public void PacketReceived(Packet packet, int size)
         {
             lock (receivedPackets)
             {
-                networkDebugger?.PacketReceived(packet);
+                networkDebugger?.PacketReceived(packet, size);
```

Comment:
Recommend renaming argument to `byteSize` because `size` is too ambiguous in meaning.